index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <header>
+    <nav class="toolbar">
+      <div class="menu_main">
+        <input type="checkbox" class="toggler">
+        <div class="humbuger"></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="">Home</a></li>
+                <li><a href="">Projects</a></li>
+                <li><a href="">Contribution</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </header>
```

Comment:
- [ ] As suggested by the previous reviewer please make sure to include a link for the logo before the `nav` tag and inside the `nav` tag you only need to place an image for the menu icon or any other way you prefer to add an icon. So please remove the code inside `<div class="menu_main">` and only place the icon ✔. 
- Here's how it should look in the end:
```
<header>
     <a href="#">LOGO</a>
     <nav>
        <img src="menu-icon">
     </nav>
</header>
```
- You can download the menu icon from Figma.